src/QMCTools/Eshdf.py

```diff
@@ -0,0 +1,246 @@
+######################################################################################
+## This file is distributed under the University of Illinois/NCSA Open Source License.
+## See LICENSE file in top directory for details.
+##
+## Copyright (c) 2021 Juha Tiihonen and QMCPACK developers.
```

Comment:
I was a bit unclear here, don't add your name to the copyright.  

[license in header](https://qmcpack.readthedocs.io/en/develop/developing.html#files)

I believe this will force us to hunt you down specifically for any future license change. Our contribution guidelines should explicitly require you to assign copyright to the project.

Reply:
Okay, seems I misunderstood you at first. I don't personally care about these so much, so let me remove the name.